src/main/java/pl/lukakan/todoapp/controller/MainController.java

```diff
@@ -0,0 +1,83 @@
+package pl.lukakan.todoapp.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.lukakan.todoapp.model.Task;
+import pl.lukakan.todoapp.model.TaskStatus;
+import pl.lukakan.todoapp.repository.TaskRepository;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class MainController {
+
+    private final TaskRepository taskRepository;
+
+    @Autowired
+    public MainController(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+    }
+
+    @GetMapping("/")
+    public String homePage(Model model) {
+        List<Task> activeTasks = taskRepository.findByTaskStatusOrderByTargetDateAsc(TaskStatus.ACTIVE);
```

Comment:
Zapytania do bazy danych są kosztowne, więc jak wiemy, że potrzebujemy obie rzeczy, to lepiej byłoby to zrobić przez jedno zapytanie, a potem rozdzielić w javie.

Reply:
Ok. Dzięki dołożę w takim razie Service :)